src/telemetry/metrics.yaml

```diff
@@ -396,6 +396,40 @@ collapse:
         description: Which tier of plan the user is on [Free, Plus]
         type: string
 
+view:
```

Comment:
@rhelmer We were talking about adding a `view` event to `ctaButton` at first. I’ve opted for adding `view` as a module that works better with the more generic `useViewTelemetry`. This makes it easier to observe other elements or types of view actions if we need to do so in the future.

Does this sound reasonable, and would we need to request a data review for this event type? The type of metrics we record is not different from the other event modules.

Reply:
With our current setup of Glean modules and with respect to the already added `view` events I’ve [adjusted the metrics.yaml](https://github.com/mozilla/blurts-server/pull/4794/files#diff-b07fc1fd4cfa7508bc162a346e98d6c12ef6d3c30ec9bbda3c8e3fe1472b637aR432) to add a `view` event to `ctaButton`.